roles/kubernetes-apps/helm/templates/helm-container.j2

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+{{ docker_bin_dir }}/docker run --rm \
```

Comment:
Regarding helm client, this is an ephemeral container that will be deleted right after. we don't need to add opts for this one since it will live less than 1 second